broker-daemon/index.js

```diff
@@ -234,6 +227,18 @@ class BrokerDaemon {
     this.orderbooks.set(marketName, new Orderbook(marketName, this.relayer, this.store.sublevel(marketName), this.logger))
     return this.orderbooks.get(marketName).initialize()
   }
+
+  async validateEngines () {
+    await Promise.all(Array.from(this.engines, async ([ symbol, engine ]) => {
+      try {
+        this.logger.info(`Validating engine configuration for ${symbol}`)
+        await engine.validateNodeConfig()
+        this.logger.info(`Validated engine configuration for ${symbol}`)
+      } catch (e) {
+        await exponentialBackoff(async () => engine.validateNodeConfig(), 10, 1000, () => console.log(`Validated engine configuration for ${symbol}`))
```

Comment:
[nit] we might want to add constants for `10` and `1000` w/ documentation on the values?

```
EXPONENTIAL_BACKOFF_ATTEMPTS = 10
EXPONENTIAL_BACKOFF_DELAY = 1000
```